### PR TITLE
Fix kernel module loading part 2

### PR DESCRIPTION
**Pull request by mrIncompetent**
```release-note
NONE
```

/assign @alvaroaleman

### CODE REVIEW

File: pkg/userdata/centos/testdata/kubelet-v1.11-aws.yaml [hunk 1]
```diff
@@ -25,8 +25,10 @@ write_files:
     modprobe ip_vs_wrr
     modprobe ip_vs_sh
 
-    # nf_conntrack_ipv4 got removed in the kernel version 5.0
-    if [ "${kernel_main_version}" -le "4" ]; then
+    # nf_conntrack_ipv4 got removed in the kernel version 4.19
+    if [[ "${kernel_main_version}" -ge "5" ]] || [[ "${kernel}" == 4.19* ]] || [[ "${kernel}" == 4.20* ]]; then
```
Comment (alvaroaleman) [inline pkg/userdata/centos/testdata/kubelet-v1.11-aws.yaml]: `s/[[/{/g` Posix compliance _and_smalller userdata :)

Comment (mrIncompetent) [reply pkg/userdata/centos/testdata/kubelet-v1.11-aws.yaml]: Then the substring comparison won't work (At least i cant get them working without adding way more characters) :grimacing: 

```sh
set -euo pipefail

kernel="4.19.8"
kernel_main_version=${kernel:0:1}

echo $kernel
echo $kernel_main_version

# nf_conntrack_ipv4 got removed in the kernel version 4.19
if [ $kernel_main_version -ge "5" ] || [ $kernel == 4.19* ] || [ $kernel == 4.20* ]; then
  echo "New Kernel"
else
  echo "Old Kernel"
fi
```

```
4.19.8
4
Old Kernel
```

Comment (alvaroaleman) [reply pkg/userdata/centos/testdata/kubelet-v1.11-aws.yaml]: (still busy discussing the matter with the expert, will give feedback once the correct POSIX way is found)

Comment (thz) [reply pkg/userdata/centos/testdata/kubelet-v1.11-aws.yaml]: `[ -z "${kernel##4.19*}" -o -z "${kernel##5*}" ]`

Comment (thz) [reply pkg/userdata/centos/testdata/kubelet-v1.11-aws.yaml]: But how about a completely different way:
```
if modinfo nf_conntrack_ipv4 2& > /dev/null; then
  modprobe nf_conntrack_ipv4
else
  modprobe nf_conntrack
fi
```

**Comment by alvaroaleman** [comment]
@thz can you review and approve?

**Pull request merged**